### ISSUE TITLE
about details on training and validation

### ISSUE THREAD

**Issue by longRookie** [talshapira/BGP2Vec#2]
Hello, **is the OXI file for BGP2vec training a single file downloaded from oix-route-view, or is it a large file for all files from October to November 2018 that can be packaged into one large file?** It is understood from the paper that all the data from two months is used, but from the code, one of the parameters of BGP2Vec is the path to the OXI file, and does not provide the operation of merging multiple small files into one large file. What are the details of the implementation here?

**Besides, what is the file used for verification?** The article says that the verification data includes as-relationships data, is this part of the data also the data from October to November 2018? When dividing the training set and the test set, the training set and the test set are divided according to the relationship of one day, and finally the average daily accuracy rate. Or merge the two-month relationship into one large set, divide the training set and the test set, and get the final accuracy rate?

I'd appreciate it if you could answer.
